recipes/nseg/meta.yaml

```diff
@@ -0,0 +1,29 @@
+{% set name = "nseg" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "8db81cd4acc8ce349872e3d22a48f11b928a2578d44e3b0fb0cb1f209f1b9e17" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/jebrosen/nseg/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - nseg 2>&1 | grep -i usage
+
+about:
+  home: https://github.com/jebrosen/nseg
+  license: Public Domain
+  license_file: LICENSE
+  summary: 'Tool to filter repeat library.'
```

Comment:
A better description: "nseg identifies and masks regions of low complexity in nucleic acid sequences".

The original citation (in case it's helpful) is `Wootton, J. C. and S. Federhen (1993)`, but `nseg` has not been actively maintained since around the mid-90s. My fork is passively maintained, with non-algorithmic changes in order to continue compiling and working correctly.

Reply:
I have changed the summary according to your suggestion